content/announcements/2025-11-26-ingress-nginx-eol-and-gateway-api.md

```diff
@@ -0,0 +1,181 @@
+    - name: https
+      hostname: 'foo.example.com'
+      port: 443
+      protocol: HTTPS
+      allowedRoutes:
+        namespaces:
+          from: All
+      tls:
+        mode: Terminate
+        certificateRefs:
+          - name: gateway-tls
+```
+
+For context, cert-manager doesn't look at the hostnames on HTTPRoutes as these
+hostnames aren't meant for TLS; they are used by Gateway API controllers to know
+which listener on the Gateway should be used for each HTTPRoute.
+
+This model works only when either:
+
+1. each team owns its own Gateway (increasing cost/infra), or
+2. the implementation makes Gateways "cheap" logical objects.
+
+It does not work for the common "one-shared-Gateway + many-teams" model. There
+is no safe way today to let each team self-manage TLS, on a shared
+Gateway, without either risky wildcards or dangerous RBAC.
+
+## ListenerSet: the missing building block
+
+Gateway API intends to introduce the ListenerSet resource (currently only as the
+experimental `XListenerSet` kind) to address the self-service challenge while
+maintaining security boundaries. ListenerSet allows:
+
+- the platform team to own one shared Gateway and its infrastructure,
+- application teams to create their own listeners and TLS settings in
+  `XListenerSet` objects.
+
+This restores the separation of concerns while preserving the security
+improvements of Gateway API's design:
+
+| Resource    | Concern                                     | Owner         |
+| ----------- | ------------------------------------------- | ------------- |
+| Gateway     | infra boundary + policy                     | Platform Team |
+| ListenerSet | per-team hostname + TLS + `certificateRefs` | Developer     |
+| HTTPRoute   | routing                                     | Developer     |
+
+Functionally, for Ingress users, XListenerSet + HTTPRoute becomes the closest
+thing to functionally similar to the Ingress experience, while remaining
+Gateway-native, as shown in the following diagram:
+
+![Similarly to the previous diagram, with Ingress, App Developers can configure TLS on their own. After, with Gateway and ListenerSet, App Developers can keep the same workflow for configuring TLS without needing to involve the Cluster Operator.](/images/announcements/2025-11-26-ingress-nginx-eol-and-gateway-api/migrating-using-listenerset.svg)
+
+## cert-manager roadmap: `XListenerSet` support in Feb 2026
+
+### What we will ship
+
+cert-manager 1.20 will include, behind a feature gate, support for the
+`cert-manager.io/issuer` and `cert-manager.io/cluster-issuer` annotations on
+`XListenerSet` resources, with XListenerSet annotations overriding Gateway
+annotations, and Gateway annotations acting as the "default issuer".
+
+### Timeline
+
+- **January 2026:** Alpha builds with `XListenerSet` support. We will need your
+  help to test it out!
+- **10 February 2026:** cert-manager 1.20 with `XListenerSet` support included
+  as an experimental feature behind:
```

Comment:
I never got around to asking how you could be so confident of this date...

I'd include at least some words like "should"... or "expects to"/"is expected to".